homework/homework.go

```diff
@@ -0,0 +1,87 @@
+package main
+
+import (
+	"fmt"
+	"gocourse05/core/camera"
+	"math/rand/v2"
+)
+
+type ID interface {
+	GetId() int
+}
+
+type Metadata interface {
```

Comment:
https://go.dev/doc/effective_go#interface-names

> By convention, one-method interfaces are named by the method name plus an -er suffix or similar modification to construct an agent noun: Reader, Writer, Formatter, CloseNotifier etc.

```suggestion
type Metadater interface {
```

Reply:
перейменував